Sources/XCTest/Public/XCTestCase.swift

```diff
@@ -108,41 +110,130 @@ open class XCTestCase: XCTest {
     /// - Note: Like the designated initializer for Apple XCTest's XCTestCase,
     ///   `-[XCTestCase initWithInvocation:]`, it's rare for anyone outside of
     ///   XCTest itself to call this initializer.
-    public required init(name: String, testClosure: @escaping XCTestCaseClosure) {
```

Comment:
Explicitly overriding this initializer is quite unusual, I believe, but it seems like this has the *potential* for source breakage if somebody is overriding it. I wonder if there's anything we can do to avoid breaking change here?